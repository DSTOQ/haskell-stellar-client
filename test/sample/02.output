TransactionEnvelope
  { _transaction =
      Transaction
        { _sourceAccount =
            PublicKeyEd25519
              { _publicKeyEd25519 =
                  25557257389159558734186508066014951066665246642133907666002899819097948594332
              }
        , _fee = Fee { _fee = 500 }
        , _seqNum = SequenceNumber { _sequenceNumber = 30381378881327023 }
        , _timeBounds = Nothing
        , _memo = MemoNone
        , _operations =
            [ Operation
                { _sourceAccount = Nothing
                , _body =
                    ManageOffer
                      ManageOfferOp
                        { _selling =
                            AssetCreditAlphanum4
                              AssetCode4 { _assetCode4 = 1112818432 }
                              PublicKeyEd25519
                                { _publicKeyEd25519 =
                                    28921469860013907089888894260325844275998968287345589102846359138918054845685
                                }
                        , _buying = AssetNative
                        , _amount = 290827
                        , _price = Price { _numerator = 5025000 , _denominator = 157 }
                        , _offerId = OfferId { _offerId = 703126 }
                        }
                }
            , Operation
                { _sourceAccount = Nothing
                , _body =
                    ManageOffer
                      ManageOfferOp
                        { _selling =
                            AssetCreditAlphanum4
                              AssetCode4 { _assetCode4 = 1112818432 }
                              PublicKeyEd25519
                                { _publicKeyEd25519 =
                                    28921469860013907089888894260325844275998968287345589102846359138918054845685
                                }
                        , _buying = AssetNative
                        , _amount = 290827
                        , _price = Price { _numerator = 5125000 , _denominator = 157 }
                        , _offerId = OfferId { _offerId = 703127 }
                        }
                }
            , Operation
                { _sourceAccount = Nothing
                , _body =
                    ManageOffer
                      ManageOfferOp
                        { _selling =
                            AssetCreditAlphanum4
                              AssetCode4 { _assetCode4 = 1112818432 }
                              PublicKeyEd25519
                                { _publicKeyEd25519 =
                                    28921469860013907089888894260325844275998968287345589102846359138918054845685
                                }
                        , _buying = AssetNative
                        , _amount = 290827
                        , _price = Price { _numerator = 5225000 , _denominator = 157 }
                        , _offerId = OfferId { _offerId = 703128 }
                        }
                }
            , Operation
                { _sourceAccount = Nothing
                , _body =
                    ManageOffer
                      ManageOfferOp
                        { _selling =
                            AssetCreditAlphanum4
                              AssetCode4 { _assetCode4 = 1112818432 }
                              PublicKeyEd25519
                                { _publicKeyEd25519 =
                                    28921469860013907089888894260325844275998968287345589102846359138918054845685
                                }
                        , _buying = AssetNative
                        , _amount = 290827
                        , _price = Price { _numerator = 5325000 , _denominator = 157 }
                        , _offerId = OfferId { _offerId = 703129 }
                        }
                }
            , Operation
                { _sourceAccount = Nothing
                , _body =
                    ManageOffer
                      ManageOfferOp
                        { _selling =
                            AssetCreditAlphanum4
                              AssetCode4 { _assetCode4 = 1112818432 }
                              PublicKeyEd25519
                                { _publicKeyEd25519 =
                                    28921469860013907089888894260325844275998968287345589102846359138918054845685
                                }
                        , _buying = AssetNative
                        , _amount = 290827
                        , _price = Price { _numerator = 5425000 , _denominator = 157 }
                        , _offerId = OfferId { _offerId = 703130 }
                        }
                }
            ]
        }
  , _signatures =
      [ DecoratedSignature
          { _hint = SignatureHint "ad5ea09c"
          , _signature =
              Signature
                "d99abbc99b4e5e3b0c0ab206909aadb223d3d5e087719f355528469fbca9ee5715f8fb5c2e0400783b97be687c3a1b74aee953463ffce07eb3860c070d4b800f"
          }
      ]
  }