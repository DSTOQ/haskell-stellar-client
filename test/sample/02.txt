AAAAADiA5SMGlbTYW1+u5SVnEokvbPeqNXENvJZ+Vw6tXqCcAAAB9ABr77QAAg+vAAAAAAAAAAAAAAAFAAAAAAAAAAMAAAABQlRDAAAAAAA/8PjK6rZT6wnyO1T+ry5U8MKX0hAtMTgBIwmZKOVc9QAAAAAAAAAAAARwCwBMrOgAAACdAAAAAAAKupYAAAAAAAAAAwAAAAFCVEMAAAAAAD/w+MrqtlPrCfI7VP6vLlTwwpfSEC0xOAEjCZko5Vz1AAAAAAAAAAAABHALAE4ziAAAAJ0AAAAAAAq6lwAAAAAAAAADAAAAAUJUQwAAAAAAP/D4yuq2U+sJ8jtU/q8uVPDCl9IQLTE4ASMJmSjlXPUAAAAAAAAAAAAEcAsAT7ooAAAAnQAAAAAACrqYAAAAAAAAAAMAAAABQlRDAAAAAAA/8PjK6rZT6wnyO1T+ry5U8MKX0hAtMTgBIwmZKOVc9QAAAAAAAAAAAARwCwBRQMgAAACdAAAAAAAKupkAAAAAAAAAAwAAAAFCVEMAAAAAAD/w+MrqtlPrCfI7VP6vLlTwwpfSEC0xOAEjCZko5Vz1AAAAAAAAAAAABHALAFLHaAAAAJ0AAAAAAAq6mgAAAAAAAAABrV6gnAAAAEDZmrvJm05eOwwKsgaQmq2yI9PV4IdxnzVVKEafvKnuVxX4+1wuBAB4O5e+aHw6G3Su6VNGP/zgfrOGDAcNS4AP
