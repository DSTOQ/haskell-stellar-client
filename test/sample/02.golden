TransactionEnvelope {transaction = Transaction {sourceAccount = PublicKeyEd25519 {unPublicKeyEd25519 = 25557257389159558734186508066014951066665246642133907666002899819097948594332}, fee = Fee {unFee = 500}, seqNum = SequenceNumber {unSequenceNumber = 30381378881327023}, timeBounds = Nothing, memo = MemoNone, operations = VarLen {unVarLen = [Operation {sourceAccount = Nothing, body = ManageOffer (ManageOfferOp {selling = AssetCreditAlphanum4 (AssetCode4 {unAssetCode4 = 1112818432}) (PublicKeyEd25519 {unPublicKeyEd25519 = 28921469860013907089888894260325844275998968287345589102846359138918054845685}), buying = AssetNative, amount = 290827, price = Price {numerator = 5025000, denominator = 157}, offerId = OfferId {unOfferId = 703126}})},Operation {sourceAccount = Nothing, body = ManageOffer (ManageOfferOp {selling = AssetCreditAlphanum4 (AssetCode4 {unAssetCode4 = 1112818432}) (PublicKeyEd25519 {unPublicKeyEd25519 = 28921469860013907089888894260325844275998968287345589102846359138918054845685}), buying = AssetNative, amount = 290827, price = Price {numerator = 5125000, denominator = 157}, offerId = OfferId {unOfferId = 703127}})},Operation {sourceAccount = Nothing, body = ManageOffer (ManageOfferOp {selling = AssetCreditAlphanum4 (AssetCode4 {unAssetCode4 = 1112818432}) (PublicKeyEd25519 {unPublicKeyEd25519 = 28921469860013907089888894260325844275998968287345589102846359138918054845685}), buying = AssetNative, amount = 290827, price = Price {numerator = 5225000, denominator = 157}, offerId = OfferId {unOfferId = 703128}})},Operation {sourceAccount = Nothing, body = ManageOffer (ManageOfferOp {selling = AssetCreditAlphanum4 (AssetCode4 {unAssetCode4 = 1112818432}) (PublicKeyEd25519 {unPublicKeyEd25519 = 28921469860013907089888894260325844275998968287345589102846359138918054845685}), buying = AssetNative, amount = 290827, price = Price {numerator = 5325000, denominator = 157}, offerId = OfferId {unOfferId = 703129}})},Operation {sourceAccount = Nothing, body = ManageOffer (ManageOfferOp {selling = AssetCreditAlphanum4 (AssetCode4 {unAssetCode4 = 1112818432}) (PublicKeyEd25519 {unPublicKeyEd25519 = 28921469860013907089888894260325844275998968287345589102846359138918054845685}), buying = AssetNative, amount = 290827, price = Price {numerator = 5425000, denominator = 157}, offerId = OfferId {unOfferId = 703130}})}]}}, signatures = VarLen {unVarLen = [DecoratedSignature {hint = SignatureHint {unSignatureHint = 2908659868}, signature = Signature {unSignature = VarLen {unVarLen = "\217\154\187\201\155N^;\f\n\178\ACK\144\154\173\178#\211\213\224\135q\159\&5U(F\159\188\169\238W\NAK\248\251\\.\EOT\NULx;\151\190h|:\ESCt\174\233SF?\252\224~\179\134\f\a\rK\128\SI"}}}]}}